lib/actions/BackendAction.js

```diff
@@ -120,6 +127,11 @@ class BackendAction {
         fsEx.outputJsonSync(path.join('', config.path, 'frontend', 'config.json'), extConfig.frontend, {spaces: 2})
         logger.info(`Updated extension config for frontend ${config.file.id}`)
       }
+
+      if (!extConfig.frontend && !extConfig.backend) {
```

Comment:
what is extConfig.frontend if not the theme config?
I find it kind of dangerous that we assume that if the extConfig object doesnt include frontend or backend that it MUST be a theme